documentation/assemblies/assembly-kafka-connect-authorization.adoc

```diff
@@ -0,0 +1,34 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-connect.adoc
+// assembly-deployment-configuration-kafka-connect-s2i.adoc
+
+[id='assembly-kafka-connect-authorization-{context}']
+
+= Kafka Connect user authorization
+
+If authorization is enabled for Kafka Connect, the `KafkaUser` resource for the Kafka Connect user must be configured to provide read/write access rights to the consumer group and
+internal topics, which are automatically configured or specified as configuration parameters in the `spec` for the `KafkaConnect` or `KafkaConnectS2I` configuration.
```

Comment:
You could give an example of an internal topic.

Reply:
How exactly are they _automatically_ configured?

Is the example below only for illustration purposes? In other words, what do we expect the user to do after reviewing the example?

Reply:
These properties are described earlier in the chapter, but I've tried to make clearer here.

> The properties for the consumer groups and internal topics are are automatically configured by {ProductName}, or they can be specified explicitly in the `spec` for the `KafkaConnect` or `KafkaConnectS2I` configuration.
> 
> The following example shows the configuration of the properties in the `KafkaConnect` resource.
> 
> The example shows the config for the internal topics and consumer groups we're talking about.